builtin/credential/aws/path_role_test.go

```diff
@@ -986,6 +989,90 @@ func TestAwsVersion(t *testing.T) {
 	}
 }
 
+// This test was used to reproduce https://github.com/hashicorp/vault/issues/7418
+// and verify its fix.
+// Please run it at least 3 times to ensure that passing tests are due to actually
```

Comment:
Can we loop this automatically?

Reply:
Errrr, if I make edits to this PR, it will diverge from what we have in master. This PR already underwent review in #7922 . I see that in our dev cycle doc, we have the following notes:

- Submit a pull request to current release branch with a title prefixed “Backport <current version>: <description of change>“
- The review process should be the same as the review process for the merge into master.  Backporting is not automatically approved if it is approved into master.

@chrishoffman or @briankassouf can you elaborate on whether this is essentially a second full review? Or is it more intended to validate the decision of whether to include something in a patch release?

Reply:
@tyrannosaurus-becks I've made the same mistake before reviewing the backport.  Everything should go through master if it needs to be reviewed again and this is more for whether to include it in a point release or not.  There are some cases that you need to change the code to support the release branch but this does not appear to be one of them.